Tweak ping device_tracker docs

ronaldevers opened the pull request:
## Proposed change

I've clarified the ping integration docs a little bit.  I could not figure out from the `device_tracker` example and the text that `hostname` (under `hosts`) was meant to be the device name. Also, the type is wrong, `hosts` is not a list but a map.

So based on the current docs I was trying to do this in my config which would not work:

```
device_tracker:
  - platform: ping
    hosts:
      - hostname: 192.168.2.10
      - hostname: 192.168.2.11
```

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

ronaldevers committed:
update ping device_tracker example and fix variable type

I've clarified the ping integration docs a little bit. I could not figure out
from the `device_tracker` example and the text that `hostname` (under `hosts`)
was meant to be the device name instead of the actual string `hostname`. Also,
the type is wrong, `hosts` is not a list but a map.

frenck commented on source/_integrations/ping.markdown:
```diff
@@ -86,14 +86,15 @@ To use this presence detection in your installation, add the following to your `
     hosts:
-      hostname: 192.168.2.10
+      hostname1: 192.168.2.10
+      hostname2: 192.168.2.11
```
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters that they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

```suggestion
```

ronaldevers replied:
Reading the docs again is says this is a map of device name to hostname or IP address. So maybe instead we make it:

```
hosts:
    device_name_1: 192.168.2.10
```

What do you think?

ronaldevers replied:
Updated and added one more commit to fix the link to the people-tracking documentation.

ronaldevers committed:
fix link for instructions for people tracking in ping docs

The device tracker page does not have any instructions for configuring the
people to be tracked. The person integration page does.

frenck reviewed (approved):
Thanks, @ronaldevers 👍 

Happy New Year 🥂
